src/data/gateway-data-source.test.ts

```diff
@@ -15,82 +15,83 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+import { strict as assert } from 'node:assert';
+import { afterEach, before, beforeEach, describe, it, mock } from 'node:test';
 import axios from 'axios';
-import chai, { expect } from 'chai';
-import sinon, { SinonSandbox } from 'sinon';
-import sinonChai from 'sinon-chai';
 import * as winston from 'winston';
 import { GatewayDataSource } from './gateway-data-source.js';
 
-chai.use(sinonChai);
+let log: winston.Logger;
+let dataSource: GatewayDataSource;
+let mockedAxiosInstance: any;
 
-describe('GatewayDataSource', function () {
```

Comment:
native testrunner doesn't support describe blocks?

Reply:
It supports, it's down below on line 59